pyiron/vasp/base.py

```diff
@@ -474,10 +474,34 @@ def collect_errors(self):
                 lines = f.readlines()
             # If the wrong convergence algorithm is chosen, we get the following error.
             # https://cms.mpi.univie.ac.at/vasp-forum/viewtopic.php?f=4&t=17071
-            for l in lines:
-                if "WARNING in EDDRMM: call to ZHEGV failed, returncode =" in l:
-                    self.status.not_converged = True
-                    break
+            if not self.get_eddrmm() == "ignore":
+                for l in lines:
+                    if "WARNING in EDDRMM: call to ZHEGV failed, returncode =" in l:
+                        if self.get_eddrmm() == "not_converged":
+                            self.status.not_converged = True
+                            break
+                        elif self.get_eddrmm() == "restart":
+                            self.status.not_converged = True
+                            if not self.input.incar["ALGO"].lower() == "normal":
+                                ham_new = self.copy_hamiltonian(self.name + "_normal")
+                                ham_new.input.incar["ALGO"] = "Normal"
+                                ham_new.set_eddrmm()
+                                ham_new.run()
+                            break
+
+    def copy_hamiltonian(self, job_name):
+        """
+        Copies a job to new one with a different name.
+
+        Args:
+            job_name (str): Job name
+
+        Returns:
+            vasp.vasp.Vasp instance: New job
```

Comment:
`pyiron.vasp.vasp.Vasp: New job` instead of  `vasp.vasp.Vasp instance: New job`. This will render the class path correctly in the docstrings!

Reply:
agreed!  Thy type `vasp.vasp.Vasp instance` is several times written in the file (like [here](https://github.com/pyiron/pyiron/blob/1fb11e4c2c2a861a62c149cd1bb182afe4ee1027/pyiron/vasp/base.py#L1275), [here](https://github.com/pyiron/pyiron/blob/1fb11e4c2c2a861a62c149cd1bb182afe4ee1027/pyiron/vasp/base.py#L1303) or [here](https://github.com/pyiron/pyiron/blob/1fb11e4c2c2a861a62c149cd1bb182afe4ee1027/pyiron/vasp/base.py#L1373)). Maybe we should open a new branch to fix it?